_samples/1_server.md

```diff
@@ -14,8 +15,9 @@ server = HTTP::Server.new do |context|
   context.response.print "Hello world, got #{context.request.path}!"
 end
 
-puts "Listening on http://127.0.0.1:8080"
-server.listen(8080)
+address = server.bind(8080)
+puts "Listening on http://#{address}"
 
-# (This example can't be run from carc.in)
```

Comment:
I'm not sure about the playground at this point. There are a couple of issues with it.
Anyway, it's not implemented yet. It might be added or not. That doesn't matter though. If we add playground integration, there simply won't be a run button for examples that don't work.

This comment doesn't doesn't help much either way and has a chance of confusing readers because it's not clear what it means. I think it's better to drop it.

Reply:
fair enough